### ISSUE TITLE
name of edit button need to be changed

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#1038]
At present:

In tracim we have button:
edit <=> editer

Expected

- [ ] name of button need to be: modify <=> modifier

**Comment by Skylsmoi** [comment]
I agree with the "Éditer", I disagree with "Modify". I'd say Edit => Modifier is the best

**Comment by PhilippeAccorsi** [comment]
This issue is from @lebouquetin .
If we need discuss about that, add need discuss in label.

**Comment by lebouquetin** [comment]
In fact:

html-documents :  edit (in english) -> modifier (en français)
files : update or upgrade ? (in english) -> mettre à jour (en français)
